dapr-http-client/src/Dapr/Client/HttpClient/Types/Core.hs

```diff
@@ -13,6 +13,8 @@ data DaprConfig = DaprConfig
   }
   deriving (Show)
 
+newtype App = App {getId :: Text}
```

Comment:
`App` is too general. Consider `DaprServerApp` or `DaprHostApp`?

Reply:
Yeah, makes sense. Changed it to `DaprHostApp`